rust/processor/src/models/token_v2_models/v2_token_datas.rs

```diff
@@ -222,14 +192,14 @@ impl TokenDataV2 {
                         token_standard: TokenStandard::V1.to_string(),
                         is_fungible_v2: None,
                         transaction_timestamp: txn_timestamp,
-                        decimals: 0,
+                        decimals: Some(0),
```

Comment:
do you want to make this always null too?

Reply:
yeah